src/test/resources/application.yml

```diff
@@ -131,6 +136,9 @@ akhq:
           - topic/insert
           - topic/delete
           - registry/version/delete
+        attributes:
```

Comment:
The same above

Reply:
@LittleWat - I added these as part of trying to locally test the changes for non-default groups. No harm in leaving it i think

Reply:
@meeraj257  
> No harm in leaving it i think

Thank you for your reply! I think so too but I just felt a bit strange that `no-filter` group has the filter(`acls-filter-regexp`).